### DISCUSSION TITLE
Garip Olan

### DISCUSSION THREAD

**Discussion by omersalih177** [sonsuzus/sonsuzus#493] [Soru Cevap]
def find_ benzersiz_değer(a, b, c):
      # Eğer a=b ise c farklıdır.
    if a == b:
        return c
    # Eğer a=c ise b farklıdır.
    elif a == c:
        return b
         # Eğer b=c ise a farklıdır.
    elif b == c:
        return a

      # Örnek:
a, b, c = 5,5,7
benzersiz_değer = find_benzersiz_değer(a, b, c)
print("Tekrar etmeyen değer:", benzersiz_değer)
Bu kod ikisinin eşit olduğu ve üçüncüsünün farklı oldugu durumu kontrol eder. find_benzersiz_değer fonksiyonu bu durumu değerlendirerek sonucu döner.

Örnek, a = 5 b = 5 c = 7 ise çıktı 7 olacaktır

**Comment by sonsuzus** [comment]
@omersalih177 önce soruyu ekle bir yazı olarak. Sonra çözüm kodunu bir cevap olarak ekle. Cevabı ekledikten sonra da kod bloğuna al.
